cime_config/machines/cmake_macros/intel_cori-haswell.cmake

```diff
@@ -1,5 +1,3 @@
-set(ALBANY_PATH "/global/cfs/cdirs/e3sm/software/albany-trilinos/albany-install-2020-08-07")
-string(APPEND FFLAGS " -convert big_endian -assume byterecl -ftz -traceback -assume realloc_lhs -fp-model consistent -fimf-use-svml")
```

Comment:
@matthewhoffman - did you intend to remove the FFLAGS line here? I understand getting rid of the ALBANY_PATH but want to make sure about the rest of it before merging

Reply:
@jonbob , I have pushed an update that fixes this inadvertent deletion (as discussed at https://github.com/MALI-Dev/E3SM/pull/16#discussion_r788184511).  Thanks for catching that!